PepMorales/PepMorales-Zaragoza.md

```diff
@@ -0,0 +1,8 @@
+## Zaragoza :heart:
```

Comment:
Yes
the typical food is a bit tricky... I don't know how to translate those dishes into English! :smile: